src/themes/themes/default.js

```diff
@@ -0,0 +1,7 @@
+import { grey, grey70, white, grey10 } from '../colours/colours';
```

Comment:
how will you export it from the lib?

Reply:
will I be able to import it as follows:
```
import { DefaultTheme } from '@livechat/design-system';

const { colors } = DefaultTheme;
const color = colors.text01;
```
or maybe you are planning something else? ;D